README.md

```diff
@@ -299,6 +299,39 @@ const MyStyledDiv = glamorous.div(
 <MyStyledDiv /> // styles applied: {padding-top: 1, padding-right: 2, padding-bottom: 3, padding-left: 4} and anything coming from `extra-thing`.
 ```
 
+##### withProps
```

Comment:
It follows similar patterns as in librarys like ImmutableJS, where you use the core API to _create the subject_, and you can build changes from working against the _subject's API_ itself, and not necessarily on the core's factory API

```js
var map = Immutable.Map({foo: 'bar'}) // factory API
var map2 = map.set('bar', 'foo') // subject API
```

So, similarly:
```js
var Component = glamorous.div()
var Component2 = Component.withProps({foo: 'bar'})
```